apps/rush-lib/src/scripts/install-run-rush.ts

```diff
@@ -49,13 +49,15 @@ function run(): void {
     ...packageBinArgs /* [build, --to, myproject] */
   ]: string[] = process.argv;
 
+  const scriptName = path.basename(scriptPath);
+  const bin = scriptName.toLowerCase() === 'install-run-rushx.js' ? 'rushx' : 'rush';
   if (!nodePath || !scriptPath) {
     throw new Error('Unexpected exception: could not detect node path or script path');
   }
 
   if (process.argv.length < 3) {
-    console.log('Usage: install-run-rush.js <command> [args...]');
-    console.log('Example: install-run-rush.js build --to myproject');
+    console.log('Usage: ${scriptName} <command> [args...]');
```

Comment:
These need backticks to make the variable substitution work:
```suggestion
    console.log(`Usage: ${scriptName} <command> [args...]`);
```